### PR TITLE
Add template folder for T4C 6.0.0

### PR DESCRIPTION
**Pull request by amolenaar**

### CODE REVIEW

**Comment by MoritzWeber0** [comment]
Have you tried testing if the build is successful with version 6.0.0?

**Comment by amolenaar** [comment]
Yes it worked fine. Only I found an issue where all my images ended up being Capella 6.0. I'm fixing that and add it to this PR.

File: capella/download_archive.py [hunk 1]
```diff
@@ -25,8 +25,11 @@ def get_directory_structure(url: str) -> list[str]:
...
 
 if __name__ == "__main__":
+    desired_version = sys.argv[1]
+    print(f"Installing Capella {desired_version}")
+
     versions = get_directory_structure(CAPELLA_INDEX_URL)
-    if not (capella_version := os.getenv("CAPELLA_VERSION")) and versions:
+    if not (capella_version := desired_version) and versions:
         capella_archive_path = versions[-1]
     else:
```
Comment (MoritzWeber0) [inline capella/download_archive.py:34]: When you have the default value set in the `Dockerfile`, we should remove the fallback here.

Comment (amolenaar) [reply capella/download_archive.py]: Indeed. I just "fixed" it.

Now it's "fixed and improved" :smile:

**Review by MoritzWeber0 — APPROVED** [review]

**Pull request merged**